Releasing.md

```diff
@@ -159,14 +163,40 @@ For the body of the post, just copy this checklist and again replace all occurre
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Ensure that the bundle files are updated to include any changes to the release branch by running <code>npm run bundle</code> in gutenberg-mobile release branch and committing any changes. </p>
+<p>o Ensure that the bundle files are updated to include any changes to the release branch by running <code>npm run bundle</code> in gutenberg-mobile release branch and committing any changes.</p>
 <!-- /wp:paragraph -->
 
+<!-- wp:paragraph -->
+<p>o <a href="https://github.com/wordpress-mobile/gutenberg-mobile/releases/new?tag=vX.XX.X&amp;target=release/X.XX.X&amp;title=Release%20X.XX.X">Create a new gutenberg-mobile GitHub Release</a>. Include a list of changes in the Release description. Ensure the checkmark "Set as the latest release" is checked, and <strong>publish the release with the "Publish release" button.</strong></p>
+<!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o <a href="https://github.com/wordpress-mobile/gutenberg-mobile/releases/new?tag=vX.XX.X&amp;target=release/X.XX.X&amp;title=Release%20X.XX.X">Create a new gutenberg-mobile GitHub Release</a>. Include a list of changes in the Release description. Ensure the checkmark next "Set as the latest release" is checked, and publish the release with the "Publish release" button.</p>
+<p>o Wait until all CI jobs for the published tag finish and succeed.</p>
```

Comment:
Although this step is implicit, I added it because it's important to wait for all jobs before updating the references in the host apps.

Reply:
I agree that there's value in adding this step here. Similarly, in the `Create the Release` section, we have a couple of tasks that relate to waiting for the Android `Publish to S3` test before creating the Android build.

**Line 64:**

```
<strong>Note:</strong> You might want to wait a bit before confirming WPAndroid PR creation so gutenberg-mobile can have enough time to finish the <code>Build Android RN Bridge &amp; Publish to S3</code> job on CI which is needed by WPAndroid CI.
```

**Line 86:**

```
<p>o Verify the WPAndroid and iOS PR builds succeed. For WPAndroid, if the PR CI tasks include a 403 error related to an inability to resolve the <code>react-native-bridge</code> dependency, you must wait for the <code>Build Android RN Bridge &amp; Publish to S3</code> task to succeed in gutenberg-mobile and then restart the WPAndroid CI tasks.</p>
```

What do you think about simplifying and adding details about the `Publish to S3` requirement for iOS as part of this PR? I'm thinking we could perhaps remove the note at line 64 and update line 86 to something like the following:

```
<p>o Verify the WPAndroid and iOS PR builds succeed. For WPAndroid, if the PR CI tasks include a 403 error related to an inability to resolve the <code>react-native-bridge</code> dependency, you must wait for the <code>Build Android RN Bridge &amp; Publish to S3</code> task to succeed in gutenberg-mobile and then restart the WPAndroid CI tasks. Similarly, for iOS, you must wait for the <code>Build iOS RN XCFramework  &amp; Publish to S3</code> task to succeed.</p>
```

Reply:
Suggestion applied in https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/143/commits/7f529f90cebe96032d6c2f3658302006e0836014.